context/app/static/js/components/tutorials/DatasetSearchPrompt/style.js

```diff
@@ -32,6 +32,7 @@ const StyledInfoIcon = styled(InfoIcon)`
 `;
 
 const StyledButton = styled(Button)`
+  color: #fff !important; // Safari sees the color, but doesn't always respect it unless forced.
```

Comment:
```suggestion
```

Reply:
We shouldn't need this at all. Setting the color prop to primary should also set the text color white. I can't reproduce the dark text in my safari browser, could you check if removing this fixes the issue?

Reply:
My mistake for adding the css in the first place.